app/js/containers/Seller/3_SellerCurrencyContainer.jsx

```diff
@@ -17,53 +18,54 @@ const CURRENCY_DATA = [
   {id: 'KRW', label: 'South Korean Won - KRW'}
 ];
 
-class SellerFiatContainer extends Component {
+class SellerCurrencyContainer extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      fiat: props.fiat
+      currency: props.seller.currency
     };
-    this.validate(props.fiat);
+    this.validate(props.seller.currency);
     this.props.footer.onPageChange(() => {
-      props.setFiatCurrency(this.state.fiat);
+      props.setCurrency(this.state.currency);
     });
   }
 
-  validate(fiat) {
-    if (!fiat.id) {
+  validate(currency) {
+    if (!currency) {
```

Comment:
Is `currency` just a string now?

Reply:
yes

Reply:
So we match the currency string with the string using the array above to get the symbol and other stuff I guess?

Reply:
Yes, the idea is that what is in local state will be in redux state and submitted later to the contract.